tests/models.py

```diff
@@ -22,11 +22,13 @@ class Meta(TreeNodeModel.Meta):
 
```

Comment:
Could you add 2 new models/test cases (instead of changing the existing one) to grant backward compatibility and be sure to cover all the conditions added to the model?